asset/css/doc.css

```diff
@@ -108,6 +110,10 @@ div.odoc .anchored:target a {
   color: rgb(204, 48, 0);
 }
 
+div.odoc *[id] {
+  scroll-margin-top: 2rem;
```

Comment:
TIL! I didn't know about scroll margins!

Reply:
There's always new features appearing, and sometimes they even become mainstream enough to use. :grin: